app/code/community/TIG/MyParcel2014/Helper/Data.php

```diff
@@ -342,18 +342,7 @@ public function getBarcodeUrl($barcode, $destination = false, $lang = false, $fo
             )
         ) {
             $barcodeUrl = self::POSTNL_TRACK_AND_TRACE_NL_BASE_URL
-                . '&b=' . $barcode;
-            /**
-             * For dutch shipments add the postcode. For international shipments add an 'international' flag.
-             */
-            if (!empty($postcode)
-                && !empty($countryCode)
-                && $countryCode == 'NL'
-            ) {
-                $barcodeUrl .= '&p=' . $postcode;
-            } else {
-                $barcodeUrl .= '&i=true';
-            }
+                . '/' . $barcode . '/' . $postcode . '/' . $countryCode;
```

Comment:
```suggestion
                . "/$barcode/$postcode/$countryCode";
```
Dit zou ook moeten werken

Reply:
dan kan het waarschijnlijk ook op 1 line of niet?
(wel even testen :p)

Reply:
Done!